projects/2020_07_25_tokenstrike.md

```diff
@@ -0,0 +1,164 @@
+# TokenStrike
+* Project Name: tokenstrike
+* Contact Email: cjd@cjdns.fr
+* Project participants:
+  * Caleb James DeLisle will work 20% of fulltime
+  * Ben Le will work 20% of fulltime
+  * Miracle Attainment Employee will work 90% of fulltime (likely Mitchell Clark)
+* Projected duration: 6 months
+* Projected effort: 7.2 person-months
+* Pre-project effort: 0.2 person-month
+* Requested PKT contribution: 40mn
+* PKT address to pay to: pkt1qju5ru8em5jz34r4g6htfvdp4cqjt4ey7ktmvv5
+
+## Project summary
+
+The objective of this project is to establish a protocol for the issuance and exchange of tokens. Tokens must be able to be issued and exchanged with minimal effort (gas fees needed for exhange of Ethereum tokens are unacceptable). A holder of tokens should be able to exchange them in a fully untrusted Lightning Network HTLC transaction. We relax the requirement that holders of tokens must be secure from attacks by the token issuer, and in it's place we add the requirement that any attack by the issuer of a token should be able to be proven to have taken place, such that an issuer who breaks protocol will be *discredited* and nodes will cease to interact with them until they are manually reset.
+
```

Comment:
How do we know a discredited node has been manually reset?

Reply:
The discredited node isn't reset, everyone else is in order to not consider them discredited anymore